{"name": "To check the functionality of View Page", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: chrome not reachable\n  (Session info: chrome=94.0.4606.81)\nStacktrace:\nBacktrace:\n\tOrdinal0 [0x0097BDE3+2473443]\n\tOrdinal0 [0x00916661+2057825]\n\tOrdinal0 [0x00822300+1057536]\n\tOrdinal0 [0x00817344+1012548]\n\tOrdinal0 [0x00817B36+1014582]\n\tOrdinal0 [0x00819215+1020437]\n\tOrdinal0 [0x00813386+996230]\n\tOrdinal0 [0x00823650+1062480]\n\tOrdinal0 [0x00874282+1393282]\n\tOrdinal0 [0x008655AB+1332651]\n\tOrdinal0 [0x00842104+1188100]\n\tOrdinal0 [0x00842F59+1191769]\n\tGetHandleVerifier [0x00B02266+1549718]\n\tGetHandleVerifier [0x00BAD4A7+2250711]\n\tGetHandleVerifier [0x00A0718B+521403]\n\tGetHandleVerifier [0x00A06229+517465]\n\tOrdinal0 [0x0091B79D+2078621]\n\tOrdinal0 [0x0091FB58+2095960]\n\tOrdinal0 [0x0091FC92+2096274]\n\tOrdinal0 [0x00929541+2135361]\n\tBaseThreadInitThunk [0x76BEFA29+25]\n\tRtlGetAppContainerNamedObjectPath [0x77607A7E+286]\n\tRtlGetAppContainerNamedObjectPath [0x77607A4E+238]\n\n", "trace": "  File \"C:\\Python39\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Python39\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"Feature\\steps\\StepDefination.py\", line 308, in step_impl\n    back = context.driver.find_element_by_xpath(\"//a[text()='Back']\")\n  File \"C:\\Python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 514, in find_element_by_xpath\n    return self.find_element(by=By.XPATH, value=xpath)\n  File \"C:\\Python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 1238, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\n  File \"C:\\Python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 418, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Python39\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 243, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "When User Click On Back_Button And Should Be Navigate To Share Page", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: chrome not reachable\n  (Session info: chrome=94.0.4606.81)\nStacktrace:\nBacktrace:\n\tOrdinal0 [0x0097BDE3+2473443]\n\tOrdinal0 [0x00916661+2057825]\n\tOrdinal0 [0x00822300+1057536]\n\tOrdinal0 [0x00817344+1012548]\n\tOrdinal0 [0x00817B36+1014582]\n\tOrdinal0 [0x00819215+1020437]\n\tOrdinal0 [0x00813386+996230]\n\tOrdinal0 [0x00823650+1062480]\n\tOrdinal0 [0x00874282+1393282]\n\tOrdinal0 [0x008655AB+1332651]\n\tOrdinal0 [0x00842104+1188100]\n\tOrdinal0 [0x00842F59+1191769]\n\tGetHandleVerifier [0x00B02266+1549718]\n\tGetHandleVerifier [0x00BAD4A7+2250711]\n\tGetHandleVerifier [0x00A0718B+521403]\n\tGetHandleVerifier [0x00A06229+517465]\n\tOrdinal0 [0x0091B79D+2078621]\n\tOrdinal0 [0x0091FB58+2095960]\n\tOrdinal0 [0x0091FC92+2096274]\n\tOrdinal0 [0x00929541+2135361]\n\tBaseThreadInitThunk [0x76BEFA29+25]\n\tRtlGetAppContainerNamedObjectPath [0x77607A7E+286]\n\tRtlGetAppContainerNamedObjectPath [0x77607A4E+238]\n\n", "trace": "  File \"C:\\Python39\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Python39\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"Feature\\steps\\StepDefination.py\", line 308, in step_impl\n    back = context.driver.find_element_by_xpath(\"//a[text()='Back']\")\n  File \"C:\\Python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 514, in find_element_by_xpath\n    return self.find_element(by=By.XPATH, value=xpath)\n  File \"C:\\Python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 1238, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\n  File \"C:\\Python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 418, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Python39\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 243, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1634630251083, "stop": 1634630263395}, {"name": "And Again User Click On View_Button And Navigate To View Page", "status": "skipped", "start": 1634630263396, "stop": 1634630263396}, {"name": "And Click On View_page Delete Button", "status": "skipped", "start": 1634630263396, "stop": 1634630263396}, {"name": "Then A Delete Pop_Up Should Be Open With Yes Or No Button Click On No Button", "status": "skipped", "start": 1634630263396, "stop": 1634630263396}, {"name": "Then Click On Yes Button", "status": "skipped", "start": 1634630263396, "stop": 1634630263396}, {"name": "Then Particular User Friend Entire Data Should Be Deleted And User Should Be Navigate To Share_page", "status": "skipped", "start": 1634630263396, "stop": 1634630263396}], "start": 1634630251081, "stop": 1634630263396, "uuid": "ea41836d-836e-464d-8a9e-dd9709488dc4", "historyId": "f5c244475c2a6c76cf4172d99f08c42b", "fullName": "Wowdare User Side End to End Functionality: To check the functionality of View Page", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "View_page"}, {"name": "feature", "value": "Wowdare User Side End to End Functionality"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}